web/src/features/charts/tooltips/AreaGraphTooltipHeader.tsx

```diff
@@ -42,10 +42,10 @@ export default function AreaGraphToolTipHeader(props: AreaGraphToolTipHeaderProp
             <Badge
               pillText={
                 estimatedPercentage
-                  ? i18n.t('estimation-card.aggregated_estimated.pill', {
+                  ? t('estimation-card.aggregated_estimated.pill', {
                       percentage: estimatedPercentage,
                     })
-                  : __('estimation-badge.fully-estimated')
+                  : t('estimation-badge.fully-estimated')
               }
```

Comment:
It's weird how we were mixing `i18n.t` and `__`, so it's great to get it all aligned 😅

Reply:
That was me like last week, got so annoyed by sprintf I ended up removing it everywhere 😅